components/Graph.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect } from 'react';
+import {
+  CartesianAxis,
+  CartesianGrid,
+  Legend,
+  Line,
+  LineChart,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { LegendLineGraphIcon } from '@openclimatefix/nowcasting-ui.icons.icons';
+import useSWR, { Fetcher } from 'swr';
+
+interface forecastDataPointProps {
+  target_datetime_utc: number;
+  expected_generation_kw: number;
+}
+
+interface forecastDataProps {
+  forecast_uuid: string;
+  site_uuid: string;
+  forecast_creation_datetime: number;
+  forecast_version: string;
+  forecast_values: forecastDataPointProps[];
+}
+
+interface unparsedForecastDataPointProps {
+  target_datetime_utc: string | number;
+  expected_generation_kw: number;
+}
+
+interface unparsedForecastDataProps {
+  forecast_uuid: string;
+  site_uuid: string;
+  forecast_creation_datetime: string | number;
+  forecast_version: string;
+  forecast_values: unparsedForecastDataPointProps[];
+}
+
+const fetcher: Fetcher<forecastDataProps> = async (url: string) => {
+  const tempData: unparsedForecastDataProps = await fetch(url).then((res) =>
+    res.json()
+  );
+  tempData.forecast_creation_datetime = Date.parse(
+    tempData.forecast_creation_datetime as string
+  );
+  tempData.forecast_values.map(({ target_datetime_utc }) => {
+    target_datetime_utc = Date.parse(target_datetime_utc as string);
+  });
+  return tempData as forecastDataProps;
+};
+
+const Graph = () => {
+  const { data, error, isLoading } = useSWR(
+    'http://localhost:3000/api/sites/pv_forecast/b97f68cd-50e0-49bb-a850-108d4a9f7b7e',
+    fetcher
+  );
+
+  const formatter = new Intl.DateTimeFormat(['en-US', 'en-GB'], {
+    hour: 'numeric',
+    minute: 'numeric',
+  });
+
+  const truncateDecimals = (number: number, digits: number) => {
+    var multiplier = Math.pow(10, digits),
+      adjustedNum = number * multiplier,
+      truncatedNum = Math[adjustedNum < 0 ? 'ceil' : 'floor'](adjustedNum);
+
+    return truncatedNum / multiplier;
+  };
+
+  const MAX = data
+    ? Math.max.apply(
+        Math,
+        data.forecast_values.map(
+          (obj: forecastDataPointProps) => obj.expected_generation_kw
+        )
+      )
+    : 0;
+
+  const tickArray = [0, MAX / 4, MAX / 2, (3 * MAX) / 4, MAX];
+
+  return (
+    <>
+      <div className=" w-screen h-[300px] bg-[#444444] rounded-lg">
+        <div className="flex ml-[5%] mt-[20px] mb-[20px] text-sm">
+          <LegendLineGraphIcon className="text-[#FFD053]"></LegendLineGraphIcon>
```

Comment:
Can this be self closing?
```suggestion
          <LegendLineGraphIcon className="text-[#FFD053]" />
```